src/app/events/RateLimit.ts

```diff
@@ -8,13 +8,13 @@ export default class implements DiscordEvent {
 	
 	async execute(rateLimitData: RateLimitData): Promise<any> {
 		try {
-			Log.warn(`rate limit reached timeout: ${rateLimitData.timeout}`, {
+			Log.warn(`rate limit reached timeout: ${rateLimitData.timeToReset}`, {
 				indexMeta: true,
 				meta: {
-					timeout: rateLimitData.timeout,
+					timeout: rateLimitData.timeToReset,
 					limit: rateLimitData.limit,
 					method: rateLimitData.method,
-					path: rateLimitData.path,
+					path: rateLimitData.majorParameter,
```

Comment:
maybe `url: rateLimitData.url`? but not sure if that is what it was renamed to

Reply:
I don't think url exists but I will check it out

Reply:
url do exist tiki is right. `path: rateLimitData.url`
you can checkout the bot template which has been converted to v14. [here](https://github.com/LordRanchoatos/Bot-Template)

Reply:
But when I write rateLimitData. url doesn't show up instead path does, everyone seems to be working with different code e.g mine is discord v14.7.0 the bot template is v14.0.3, I modified all necessary for the code to run, when it was on .url yarn start failed on my pc saying that property doesn't exist

Reply:
There must be something else wrong. Maybe try to delete the `dist` and `node_modules` folders to make a clean install.  In vscode if you shift + click on the `RateLimitData` type it should show you the interface. Also the event was renamed to `rateLimited`.